packages/dds/tree/src/test/shared-tree/fuzz/targetedFuzz.spec.ts

```diff
@@ -0,0 +1,120 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { strict as assert } from "assert";
+import {
+	AsyncGenerator,
+	makeRandom,
+	performFuzzActionsAsync,
+	SaveInfo,
+} from "@fluid-internal/stochastic-test-utils";
+import {
+	JsonableTree,
+	fieldSchema,
+	SchemaData,
+	rootFieldKey,
+	moveToDetachedField,
+	compareUpPaths,
+	rootFieldKeySymbol,
+	UpPath,
+} from "../../../core";
+import { FieldKinds, namedTreeSchema } from "../../../feature-libraries";
+import { brand } from "../../../util";
+import { TestTreeProvider, SummarizeType, initializeTestTree, validateTree } from "../../utils";
+import { FuzzTestState, makeOpGenerator, Operation } from "./fuzzEditGenerators";
+import { fuzzReducer } from "./fuzzEditReducers";
+import { runFuzzBatch } from "./fuzzUtils";
+
+const initialTreeState: JsonableTree = {
+	type: brand("Node"),
+	fields: {
+		foo: [
+			{ type: brand("Number"), value: 0 },
+			{ type: brand("Number"), value: 1 },
+			{ type: brand("Number"), value: 2 },
+		],
+		foo2: [
+			{ type: brand("Number"), value: 0 },
+			{ type: brand("Number"), value: 1 },
+			{ type: brand("Number"), value: 2 },
+		],
+	},
+};
```

Comment:
Now that the tests suites are broken up into two separate files, should we move the `initialTreeState` into the `fuzzUtils.ts` file and import it into both files?

Reply:
Good idea. Thanks. Done.